server/models/__init__.py

```diff
@@ -0,0 +1,24 @@
+import server.models.model
+from server.models.db import db
+from server.models.proxy import ModelProxy
+
+# The imports below are order sensitive
+from server.models.model.GradingTask import GradingTask
```

Comment:
Not only is this somewhat order sensitive, it doesn't look nice and we'll have to remember to add a line in here when creating new modules. 

There is probably an automated to do this - it would be hacky but I think it'd be better than this ...